js/components/post/form.vue

```diff
@@ -1,16 +1,17 @@
 <template>
 <div>
-<vertical-form v-ref:form :fields="fields" :model="post"></vertical-form>
+    <vform v-ref:form :fields="fields" :model="post"></vform>
```

Comment:
We are doing the opposite, ie. transform `vfrom`/`hform` into  `vertical-form` `horizontal-form` for the following reason:
- this is more explicit
- web component spec prohibe single term markup (you need a dash in a web component markup)

Reply:
argh. I was overzealous at getting a homogeneous code from the wrong example.